src/matrix_validator/validator.py

```diff
@@ -42,3 +50,31 @@ def get_output_format(self):
     def get_report_file(self):
         """Get the path to the report file."""
         return os.path.join(self.report_dir, f"report.{self.output_format}")
+
+    def get_biolink_model_prefix_keys(self):
+        """Get biolink model prefix keys."""
+        try:
+            prefixes = list(requests.get("https://w3id.org/biolink/biolink-model-prefix-map.json", timeout=10).json().keys())
```

Comment:
Is there no way to get this from the biolink model package? (Not keen on runtime web dependencies especially if we will run this is relatively contained pieces of infrastructure)

Reply:
I went back and forth with @kevinschaper on this.  I originally was pulling it down at build time, but it was from a biolink-model.yaml that was incomplete.  I then was reading the biolink_model.yaml from the bmt package...which was still incomplete.  And, finally, just pulled from the requested json for the "best" source of prefixes.

Reply:
I renamed the function.

Reply:
Ok, I made https://github.com/biolink/biolink-model/pull/1551 to copy the prefixmaps into the src, which should get them into the python environment and accessible with importlib, and Sierra made a new release https://github.com/biolink/biolink-model/releases/tag/v4.2.6-rc4 that should make the prefix json available.